docs/features/inventory.md

```diff
@@ -0,0 +1,14 @@
+# Inventory
+
+The local inventory management can help your shop in preventing back-orders or managing order quantities.
```

Comment:
"Back orders" is with a hyphen here, but later it's without it. According to Grammarly it's "Backorders" but I've seen all 3 versions on the internet, so it's up to you. 😄

Also I would recommend to replace "shop" with "store", because store is more commonly used term in American English and I think we are using that here.

Reply:
I'll use "back orders" because we already use this spelling in InventoryPart editor.

Isn't "store" only for locations where products are stored and sold physically? I think "shop" is normal for e-commerce since it's often called a "web shop" or "e-shop". Also afaik "store" is American-only, but "shop" is used by both American and British.